src/scenic/core/regions.py

```diff
@@ -1546,77 +1585,85 @@ class BoxRegion(MeshVolumeRegion):
 
     By default the unit box centered at the origin and aligned with the axes is used.
 
-    Args:
-        name: An optional name to help with debugging.
-        dimensions: An optional 3-tuple, with the values representing width, length, height respectively.
-          The mesh will be scaled such that the bounding box for the mesh has these dimensions.
-        position: An optional position, which determines where the center of the region will be.
-        rotation: An optional Orientation object which determines the rotation of the object in space.
-        orientation: An optional vector field describing the preferred orientation at every point in
-          the region.
-        tolerance: Tolerance for collision computations.
-        engine: Which engine to use for mesh operations. Either "blender" or "scad".
+    Parameters are the same as `MeshVolumeRegion`, with the exception of the ``mesh``
+    parameter which is excluded.
     """
-    def __init__(self, name=None, dimensions=None, position=None, rotation=None, orientation=None,
-                 tolerance=1e-8, onDirection=None, engine="blender"):
+    def __init__(self, **kwargs):
```

Comment:
You'll need to have `*args` here too unless you want to make the arguments to `MeshVolumeRegion` keyword-only. (There are enough of them that it's probably a good idea, especially since most people will just use `fromFile` anyway.)

Reply:
Ahh good point, I'll add an args to be safe.